cwms-http-client/src/main/java/mil/army/usace/hec/cwms/http/client/CwmsHttpLoggingInterceptor.java

```diff
@@ -78,4 +86,9 @@ void setLogLevel(Level level) {
         updateInterceptorLogLevel();
     }
 
+    void logStackTraceForRequest(Request request) {
+        if (LOGGER.isLoggable(Level.FINEST)) {
+            LOGGER.log(Level.FINEST, new Exception(), () -> "CWMS HTTP API executing request: " + request.url());
```

Comment:
I suggest that the exception be given a message such as "CurrentStackTrace"

Reply:
Updated to add message